JS/main.js

```diff
@@ -0,0 +1,49 @@
+let myBooks = [];
+
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+}
+
+function getBooks() {
+  myBooks = JSON.parse(localStorage.myBooks);
+  const books = document.getElementById('books');
+  books.innerHTML = '';
+  let id = 0;
+
+  myBooks.forEach((book) => {
+    books.innerHTML += '<li>'
+    + `<h3>${book.title}</h3>`
+    + `<p>${book.author}</p>`
+    + `<button onClick="destroyBook(${id += 1})">Remove</button>`
+    + '</li>';
+  });
+}
+
+function updateLocalStorage() {
+  localStorage.myBooks = JSON.stringify(myBooks);
+  getBooks();
+}
+
+function newBook(e) {
+  e.preventDefault();
+  const title = document.getElementById('title');
+  const author = document.getElementById('author');
+  myBooks.push(new Book(title.value, author.value));
+  updateLocalStorage();
+  title.value = '';
+  author.value = '';
+}
+
```

Comment:
- [ ] It should have a function that removes a book from the collection.
For Example: 
```
const books = [];

const removeBook = (bookID) => {
  // bookID can be any other property, e.g. title or author, in as much it is unique
  books = books.filter(({ id }) => bookID !== id); // This assumes that `bookID` parameter is the book id
};
```

Reply:
Thanks for the feedback. On line 45 we have a `destroyBook` function that removes a book from a collection, is that what you are asking for?